src/mixed-line-bar-chart/__integ__/mixed-line-bar-chart.test.ts

```diff
@@ -441,6 +442,25 @@ describe('Details popover', () => {
     })
   );
 
+  describe('keeps the popover position when it resizes due to interacting with the popover itself', () => {
+    test.each(['hover', 'click', 'keyboard'])('Interaction type: %s', interactionType =>
+      setupPopoverPositionTest(async page => {
+        await page.setWindowSize({ width: 900, height: 500 });
+        expect(page.findDetailPopover()).toBeTruthy();
+        await page.focusCoordinate(0, interactionType);
```

Comment:
nit: The name of the utility is a little confusing. How about something like: `page.openPopoverOnBarGroup(0, interactionType)` ?

Reply:
Changed in the last commit. Also removed the previous line which didn't make sense, and was passing only because test util functions like `wrapper.findDetailPopover` returns a wrapper in integration tests even if the element isn't found.